sdk/webpubsub/azure-messaging-webpubsubservice/setup.py

```diff
@@ -64,7 +64,7 @@
     install_requires=[
         "isodate<1.0.0,>=0.6.1",
         "azure-core<2.0.0,>=1.24.0",
-        "pyjwt>=1.7.1",
+        "pyjwt>=2.0.0"
     ],
     python_requires=">=3.7",
```

Comment:
```suggestion
    python_requires=">=3.8",
```
Python SDK doesn't support 3.7 anymore